Modules/_ctypes/_ctypes.c

```diff
@@ -5582,6 +5582,12 @@ PyInit__ctypes(void)
 
     PyModule_AddObject(m, "RTLD_LOCAL", PyLong_FromLong(RTLD_LOCAL));
     PyModule_AddObject(m, "RTLD_GLOBAL", PyLong_FromLong(RTLD_GLOBAL));
+#if HAVE_DECL_RTLD_MEMBER
+    PyModule_AddObject(m, "RTLD_MEMBER", PyLong_FromLong(RTLD_MEMBER));
+#endif
+#if HAVE_DECL_RTLD_NOW
+    PyModule_AddObject(m, "RTLD_NOW", PyLong_FromLong(RTLD_NOW));
+#endif
```

Comment:
Please move this to Modules/posixmodule.c. We already have many os.RTLD_* constants there... like os.RTLD_NOW ;-)

Reply:
ok. as above. In my previous convos about this - they were being put into _ctypes - so I just used that convention.